tests/copy-paste.js

```diff
@@ -457,7 +457,7 @@ define([
                 ["id", "type", "calculateAttr"],
                 ["/radius", "DataBindOnly", "42"],
                 ["/pi", "DataBindOnly", "3.1415"],
-                ["/circumference", "DataBindOnly", "2 * #form/pi * #form/radius"],
+                ["/circumference", "DataBindOnly", "2 * `#form/pi` * `#form/radius`"],
```

Comment:
Seems semi-likely that humans may edit this format (in Excel), and it would be nice to not make them jump through more syntax hoops.